SurfaceTopography/UniformLineScanAndTopography.py

```diff
@@ -586,23 +616,12 @@ class DetrendedUniformTopography(DecoratedUniformTopography):
     to the topography data to extract trend lines. The resulting topography
     is then detrended by substracting these trend lines.
 
-    Note on periodicity: Detrended Topographies with mode other than "center"
+    Note on periodicity: Detrended Topographies with mode other than 'center'
     will have `is_periodic` property set to False.
-
```

Comment:
Where does the detrend_mode list end up now ?

Reply:
That is a good question. Sphinx does not appear to show the docstring of the constructor. Maybe we need to simply place the docstring of the constructor into the class. Not sure whether there is a sphinx option that changes this default behavior.

Reply:
I've convinced sphinx to include `__init__` into the documentation